sdk/core/azure-core/azure/core/messaging.py

```diff
@@ -0,0 +1,118 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import datetime as dt
+import uuid
+import base64
+
+try:
+    from typing import TYPE_CHECKING
+except ImportError:
+    TYPE_CHECKING = False
+
+if TYPE_CHECKING:
+    from typing import Any, Dict
+
+__all__ = ["CloudEvent"]
+
+def _decode(content):
+    try:
+        return base64.b64decode(content.encode('utf-8'))
```

Comment:
- ascii is enough, per base64 spec. 
- I'm not even sure we need to encode, could you describe the scenario where we do?

Reply:
Yeah, this method is confusing to me. Is the only reason for it to exist to normalize the exception type?

Reply:
yes - thinking about it, i think we dont need this method at all - removing it